evutil.c

```diff
@@ -3030,3 +3030,124 @@ evutil_free_globals_(void)
 	evutil_free_secure_rng_globals_();
 	evutil_free_sock_err_globals();
 }
+
+int 
+evutil_set_tcp_keepalive(evutil_socket_t fd, int on, int timeout)
+{
+    int idle;
+    int intvl;
+    int cnt;
+
+    /* Prevent compiler from complaining unused variables warnings. */
+    (void) idle;
+    (void) intvl;
+    (void) cnt;
+
+    if (timeout <= 0) return 0;
```

Comment:
Usually we do not online syntax, i.e. it should be like this (here and everywhere else):

```suggestion
    if (timeout <= 0)
        return 0;
```

Reply:
Done.